content/en/blog/2023/spring-native/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Observe your Spring Native application with OpenTelemetry instrumentation
+linkTitle: OTel Spring Native
+date: 2023-11-28
+author: >-
+  [Jean Bisutti](https://github.com/jeanbisutti) (Microsoft)
+---
+
+You may know the
+[OpenTelemetry Java agent](https://opentelemetry.io/docs/instrumentation/java/automatic/).
+However, Java agents are
+[not officially supported](https://github.com/oracle/graal/issues/1065) with
+GraalVM native images today. This post presents an OpenTelemetry solution to get
+telemetry data with Spring Boot native image applications. ## Instrumentation of
+Spring Boot native image applications - A history of the last months The
+OpenTelemetry contributors have used the existing OpenTelemetry Spring Boot
```

Comment:
```suggestion
telemetry data with Spring Boot native image applications. 

## A history of the last months 

The OpenTelemetry contributors have used the existing OpenTelemetry Spring Boot
```

The section title is too long, not only for prettier but also eventually for the TOC, keep it short.